osu.Game/Graphics/UserInterface/ShowMoreButton.cs

```diff
@@ -126,19 +131,21 @@ private class ChevronIcon : SpriteIcon
         {
             private const int icon_size = 8;
 
+            private Color4 accentColour;
+
+            public Color4 AccentColour
+            {
+                get => accentColour;
+                set => accentColour = Colour = value;
+            }
```

Comment:
Why does this exist? It seems you wanted an alias for `Colour`, but if so, this is wrong due to the fact you can do the following:
```csharp
spriteIcon.AccentColour = Color4.Black;
spriteIcon.Colour = Color4.White;
// now the two properties return different values
```